client/components/happychat/style.scss

```diff
@@ -279,7 +279,10 @@
 	display: flex;
 	flex-direction: row-reverse;
 	padding: 0 10px;
-	margin: 6px 0;
+	margin-top: 6px;
```

Comment:
Yep — but I intentionally moved away from shorthand, as the values are overridden later on — and though "long-hand" should still be able to override shorthand, something about that feels wrong to me.